include/envoy/http/filter.h

```diff
@@ -829,11 +832,14 @@ class StreamEncoderFilterCallbacks : public virtual StreamFilterCallbacks {
    *                       response headers.
    * @param grpc_status the gRPC status code to override the httpToGrpcStatus mapping with.
    * @param details a string detailing why this local reply was sent.
+   * @param retain_http_status_for_grpc int, which if nonzero is used as the HTTP response
```

Comment:
I think this should say 'bool' and that it is a flag, noting that the code likely comes from the response headers.